main.js

```diff
@@ -7,6 +7,20 @@ const BrowserWindow = electron.BrowserWindow;
 const path = require('path');
 const url = require('url');
 const fs = require('fs');
+const winston = require('winston')
+
+let level = 'warn'
+if (process.env.DEBUG && process.env.DEBUG.includes('hatch-previewer'))
+  level = 'debug'
```

Comment:
4 spaces indent. might also be better with braces even if it's a single-line conditional

Reply:
I was trying to stick to the style that was in the majority of JS code, but we can certainly fix it all up at once. See comment below.